src/main/java/com/google/sps/servlets/AccountData.java

```diff
@@ -41,27 +50,41 @@ public String getLogInOutURL() {
     public String getSignUpURL() {
       return signUpURL;
     }
+
+    public UserInfo getUserInfo() {
+      return userInfo;
+    }
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     UserService userService = UserServiceFactory.getUserService();
 
-    String logInOutLink;
-    String signUpLink;
     JsonAccountInfo jsonAccountInfo;
 
     if (userService.isUserLoggedIn()) {
-      logInOutLink = userService.createLogoutURL("/index.html");
+      String logInOutLink = userService.createLogoutURL("/index.html");
 
       Entity userEntity = getUserEntity(userService.getCurrentUser().getUserId());
-
-      jsonAccountInfo = new JsonAccountInfo(true, logInOutLink, null);
+      UserInfo userInfo = new UserInfo("This", "account", "doesnt", "exist");
```

Comment:
What's this for?

Reply:
Because I have two constructors, passing in "null" causes an amibguity error, so I made an empty object so that it would call the correct constructor.

Reply:
Use a special static instance to indicate that it's invalid.